skype_sender_prod.py

```diff
@@ -36,75 +36,76 @@ def remove_html_tags(text):
 #you can setup this route in Slack but we strongly  recommend to use Corezoid instead
 @app.route('/listen/skype', methods=['GET', 'POST'])
 def listen_skype():
-     if request.method == 'POST':
-        app.logger.info('/listen request:'+ "".join( request.data.splitlines()))
-        _json =  request.get_json(force=True)
+    if request.method != 'POST':
+        return
 
-        if "challenge" in _json:
-            return _json["challenge"]
+    app.logger.info('/listen request:'+ "".join( request.data.splitlines()))
+    _json =  request.get_json(force=True)
 
+    if "challenge" in _json:
+        return _json["challenge"]
 
-
-        if _json["event"]["type"] == "message":
-            if not 'sybtype' in  _json["event"]:
-                r = requests.post(corezoidUrl,data=request.data)
-                app.logger.info('/corezoid answer:' + "".join(r.content.splitlines()))
-                return '{"OK":true}'
-
+    if _json["event"]["type"] == "message" and 'sybtype' not in _json["event"]:
+        r = requests.post(corezoidUrl,data=request.data)
+        app.logger.info('/corezoid answer:' + "".join(r.content.splitlines()))
         return '{"OK":true}'
 
+    return '{"OK":true}'
+    
+

```

Comment:
Function `listen_skype` refactored with the following changes:

- Add guard clause ([last-if-guard](https://docs.sourcery.ai/refactorings/last-if-guard/))
- Merge nested if conditions ([merge-nested-ifs](https://docs.sourcery.ai/refactorings/merge-nested-ifs/))
- Simplify logical expression using De Morgan identities ([de-morgan](https://docs.sourcery.ai/refactorings/de-morgan/))